docs/documentation/using-notify.md

```diff
@@ -48,10 +48,11 @@ Heroku.
 To save the key on Heroku, go to the settings page of your app, click
 ‘Reveal config vars’ and fill in the two textboxes like this (where
 xxxxxxx is the key you’ve copied from Notify):
-
-KEY            | VALUE
----------------|----------
-`NOTIFYAPIKEY` | `xxxxxxx`
+```
+KEY          | VALUE
+-------------|----------
+NOTIFYAPIKEY | xxxxxxx
+```
```

Comment:
![image](https://user-images.githubusercontent.com/355079/47712312-42609680-dc2f-11e8-848f-520efb231e2b.png)

Feels like the right thing to do here would have been to fix the styling of tables, rather than make a code block. Is this an easy fix I’d be able to pick up?

Reply:
hmm good point, I'm not sure how easy it is to be honest, feel free to look into it. I think the code block is ok for now?